pkgs/development/libraries/kde-frameworks/kdesu.nix

```diff
@@ -11,4 +11,5 @@ mkDerivation {
   buildInputs = [ kcoreaddons ki18n kpty kservice qtbase ];
   propagatedBuildInputs = [ kpty ];
   outputs = [ "out" "dev" ];
+  patches = [ ./kdesu-search-for-wrapped-daemon-first.patch ];
```

Comment:
Please move this file to `.../kde-frameworks/kdesu/default.nix` and update `kde-frameworks/default.nix`.

Reply:
Done.